index.es.js

```diff
@@ -1,12 +1,5 @@
-export {
-  Future,
-  Future as default,
-  isFuture,
-  reject,
-  of,
-  never,
-  isNever
-} from './src/core';
+export {default, default as Future} from './src/future';
```

Comment:
Reading through [the spec](https://tc39.github.io/ecma262/#prod-ExportDeclaration), it seems to be alright. One concerning rule is the following:

> For each *IdentifierName* `n` in ReferencedBindings of *ExportClause*: It is a Syntax Error if StringValue of `n` is a *ReservedWord*

In the [spec for *ReservedWord*](https://tc39.github.io/ecma262/#prod-ReservedWord) we see that `default` *is* actually a reserved word. This would mean that `export {Future as default} from '...'` also shouldn't work.